src/az-databricks/deploy/bicep/main.bicep

```diff
@@ -1,30 +1,39 @@
+targetScope = 'subscription'
```

Comment:
Will we need different / new AzDO pipelines and/or GitHub workflow components to deploy a Bicep file at the subscription scope?  The existing ones operate on Bicep files at the resource group scope.

Reply:
Yes, I think we would need a different one. I will ping you offline to discuss this.